src/blot-formatters/actions/image-resize-action.ts

```diff
@@ -0,0 +1,76 @@
+import { ResizeAction } from "quill-blot-formatter";
+
+/**
+ * Custom resize action to disable text selection
+ * while dragging image resize handles
+ */
+export class ImageResizeAction extends ResizeAction {
+    // Can't use super.onDrag here
+    // because it's defined as a property
+    // on the superclass, instead of as a method,
+    // so the logic is copied from the superclass here
+    onDrag = (event: MouseEvent) => {

```

Comment:
Recommend adding explicit scope (`public onDrag(...)`) and appropriate regions throughout the file